src/tests/Login.test.jsx

```diff
@@ -107,3 +107,26 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('Checking if the fields are empty', async () => {
+    render(<Login />)
+
+    fireEvent.change(screen.getByPlaceholderText('Username or Email'), {
+        target: { value: "" },
+    })
+    expect(screen.queryByLabelText("errorMessage")).toBeRequired("");
+    fireEvent.change(screen.getByPlaceholderText('Password'), {
+        target: { value: "" },
+    })
+    expect(screen.queryByLabelText("errorMessage")).toBeRequired("");
+
+    act(() => {
+        fireEvent.click(screen.getByRole('button', { name: "Login" }), {
+            target: { value: 'true' },
+        })
+    });
+    expect(screen.queryByLabelText("errorMessage")).toHaveTextContent("Please fill in the field");
```

Comment:
@khushishikhu. from the @testing-library/jest-dom .toBeRequired() I referred to you [here](https://github.com/anitab-org/bridge-in-tech-web/pull/83#discussion_r480874784) seems to be used with getByTestId. Have you tried giving a test id to the input field on the form and refer it on the test using `expect(getByTestId...` as shown on the jest-do example?